lib/features/overview/bloc/givt_bloc.dart

```diff
@@ -233,9 +233,11 @@ class GivtBloc extends Bloc<GivtEvent, GivtState> {
     Emitter<GivtState> emit,
   ) async {
     try {
+      final fromDate = '${event.year}-01-01';
+      final tillDate = '${event.year}-12-31';
+
```

Comment:
Maybe is better to use a DateTime object and a DateFormatter

Reply:
To use DateTime and a DateFormatter the lines would look like:
```
final fromDate = DateFormat('yyyy-MM-dd').format(DateTime.utc(int.parse(year), 01, 01));
final tillDate = DateFormat('yyyy-MM-dd').format(DateTime.utc(int.parse(year), 12, 31));
```
I dont really see the point of parsing the year string into and integer, to then create a dateTime object, to then format that into a string again. 
What are the benefits of using DateTime?